### PR TITLE
Remove Component namespace on Doctrine

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by loic425** [comment]
@diimpp theses classes were introduced in 1.11 alpha too

**Review by Prometee — APPROVED** [review]

**Review by oallain — APPROVED** [review]
LGFM 👍

File: src/Bundle/Resources/config/services/state.xml [hunk 1]
```diff
@@ -95,15 +95,17 @@
             <tag name="sylius.state_provider" />
         </service>
 
-        <service id="Sylius\Component\Resource\Doctrine\Common\State\PersistProcessor">
+        <service id="Sylius\Resource\Doctrine\Common\State\PersistProcessor">
             <argument type="service" id="doctrine" />
             <tag name="sylius.state_processor" />
         </service>
+        <service id="Sylius\Component\Resource\Doctrine\Common\State\PersistProcessor" alias="Sylius\Resource\Doctrine\Common\State\PersistProcessor" />
```
Comment (diimpp) [inline src/Bundle/Resources/config/services/state.xml]: Is deprecation notice needed over here?

https://symfony.com/doc/current/service_container/alias_private.html#deprecating-service-aliases

File: src/Bundle/Resources/config/services/state.xml [hunk 2]
```diff
@@ -95,15 +95,17 @@
...
 
-        <service id="Sylius\Component\Resource\Doctrine\Common\State\RemoveProcessor">
+        <service id="Sylius\Resource\Doctrine\Common\State\RemoveProcessor">
             <argument type="service" id="doctrine" />
             <tag name="sylius.state_processor" />
         </service>
+        <service id="Sylius\Component\Resource\Doctrine\Common\State\RemoveProcessor" alias="Sylius\Resource\Doctrine\Common\State\RemoveProcessor" />
```
Comment (diimpp) [inline src/Bundle/Resources/config/services/state.xml]: Is deprecation notice needed over here?

**Review by diimpp — APPROVED** [review]

File: src/Bundle/Resources/config/services/state.xml (hunk 1 above)
Comment (loic425) [reply src/Bundle/Resources/config/services/state.xml]: hum, this is not needed here, I have to remove that alias cause this is from 1.11. I forgot to remove that.

File: src/Bundle/Resources/config/services/state.xml (hunk 2 above)
Comment (loic425) [reply src/Bundle/Resources/config/services/state.xml]: thx, you have good eyes, I have to remove it too.

File: src/Bundle/Resources/config/services/state.xml (hunk 1 above)
Comment (loic425) [reply src/Bundle/Resources/config/services/state.xml]: thx for the doc, cause I will need that for future PRs soon.

**Review by diimpp — APPROVED** [review]

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @loic425!

**Pull request merged**